### DISCUSSION TITLE
web3auth connect modal loading too long in mobile

### DISCUSSION THREAD

**Discussion by aungkyaw-dz** [Web3Auth/Web3Auth#573] [Help]
I tried to use use web3auth in reactJS and its working fine on PC browser but the connect modal show only loading  in mobile browsers.

SDK version - 
"@web3auth/base": "^2.0.0",
"@web3auth/web3auth": "^2.0.0",
![IMG_4683](https://user-images.githubusercontent.com/98510096/189035506-f2e3a2ec-1217-41ec-88ef-745151016fe9.png)

**Comment by rinzler99** [comment]
Hey, 

Which browser is it?

**Reply by rinzler99** [reply]
And also ensure that you are using the latest sdk versions.
